File: Pi java/src/Pi/entities/Commande.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Pi.entities;

import java.sql.Date;
import java.time.LocalDateTime;

/**
 *
 * @author Aziz
 */
public class Commande {
    int idcommande;
    int idlivreur_id;
    String Produit;
    int Quantite;
    Date Date;
    float Total;
    int adresse_id;

    public Commande() {
    }

    public Commande(int idcommande, int idlivreur_id, String Produit, int Quantite, Date Date, float Total, int adresse_id) {
        this.idcommande = idcommande;
        this.idlivreur_id = idlivreur_id;
        this.Produit = Produit;
        this.Quantite = Quantite;
        this.Date = Date;
        this.Total = Total;
        this.adresse_id = adresse_id;
    }

    public Commande(String Produit, int Quantite, Date Date, float Total) {
        this.Produit = Produit;
        this.Quantite = Quantite;
        this.Date = Date;
        this.Total = Total;
    }

    public Commande(String Produit, int Quantite, Date Date, float Total, int adresse_id) {
        this.Produit = Produit;
        this.Quantite = Quantite;
        this.Date = Date;
        this.Total = Total;
        this.adresse_id = adresse_id;
    }

    public Commande(int idcommande, String Produit, int Quantite, float Total, int adresse_id) {
        this.idcommande = idcommande;
        this.Produit = Produit;
        this.Quantite = Quantite;
        this.Total = Total;
        this.adresse_id = adresse_id;
    }

   
   
    

    
    

    public int getIdcommande() {
        return idcommande;
    }

    public void setIdcommande(int idcommande) {
        this.idcommande = idcommande;
    }

    public int getIdlivreur_id() {
        return idlivreur_id;
    }

    public void setIdlivreur_id(int idlivreur_id) {
        this.idlivreur_id = idlivreur_id;
    }

    public String getProduit() {
        return Produit;
    }

    public void setProduit(String Produit) {
        this.Produit = Produit;
    }

    public int getQuantite() {
        return Quantite;
    }

    public void setQuantite(int Quantite) {
        this.Quantite = Quantite;
    }

    public Date getDate() {
        return Date;
    }

    public void setDate(Date Date) {
        this.Date = Date;
    }

    public float getTotal() {
        return Total;
    }

    public void setTotal(float Total) {
        this.Total = Total;
    }

    public int getAdresse_id() {
        return adresse_id;
    }

    public void setAdresse_id(int adresse_id) {
        this.adresse_id = adresse_id;
    }

    @Override
    public String toString() {
        return "Commande{" + "idcommande=" + idcommande + ", idlivreur_id=" + idlivreur_id + ", Produit=" + Produit + ", Quantite=" + Quantite + ", Date=" + Date + ", Total=" + Total + ", adresse_id=" + adresse_id + '}';
    }
    
}

File: Esports-Application/src/Entity/Commande.php
<?php

namespace App\Entity;
use Symfony\Component\Validator\Constraints as Assert;
use App\Entity\Adresse;

use App\Repository\CommandeRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Serializer\Annotation\Groups;

/**
 * @ORM\Entity(repositoryClass=CommandeRepository::class)
 */
class Commande
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     * @Groups("bawez")

     */
    private $idcommande;


    /**
     * @ORM\ManyToOne(targetEntity="App\Entity\Livreur", inversedBy="Livreur")
     * @Groups("bawez")

     */
    private    $idlivreur;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank(message="product is required")
     * @Groups("bawez")

     */
    private $Produit;

    /**
     * @ORM\Column(type="integer")
     * @Assert\NotBlank(message="Quantity is required")
     * @Groups("bawez")

     */
    private $Quantite;

    /**
     * @ORM\Column(type="float")
     * @Assert\NotBlank(message="Total is required")
     * @Groups("bawez")

     */
    private $Total;
    /**
     * @ORM\Column(type="datetime")
     * @Assert\NotBlank(message="Date is required")
     * @Groups("bawez")

     */
    protected $date;

    /**
     * @ORM\ManyToOne(targetEntity="App\Entity\Adresse", inversedBy="Commande")
     * @Groups("bawez")

     */
    private $adresse;



    public function __construct()
    {
        $this->date = new \DateTime('now');
    }

    public function __toString()
    {
        // TODO: Implement __toString() method.
        return $this->adresse;
    }

    /**
     * @return \DateTime
     */
    public function getDate(): \DateTime
    {
        return $this->date;
    }

    /**
     * @param \DateTime $date
     */
    public function setDate(\DateTime $date): void
    {
        $this->date = $date;
    }



    /**
     * @return mixed
     */
    public function getIdCommande()
    {
        return $this->idcommande;
    }

    /**
     * @param mixed $idcommande
     */
    public function setIdCommande($idcommande): void
    {
        $this->idcommande = $idcommande;

    }

    /**
     * @return mixed
     */
    public function getIdLivreur():?Livreur
    {
        return $this->idlivreur;
    }

    /**
     * @param mixed $idlivreur
     */
    public function setIdLivreur(?livreur $idlivreur): self
    {
        $this->idlivreur = $idlivreur;
        return $this;
    }

    /**
     * @return Adresse
     */
    public function getAdresse(): ?Adresse
    {
        return $this->adresse;
    }

    /**

     * @return $this
     */
    public function setAdresse(Adresse $adresse): self
    {
        $this->adresse = $adresse;

        return $this;
    }


    /**
     * @return mixed
     */
    public function getProduit()
    {
        return $this->Produit;
    }

    /**
     * @param mixed $Produit
     */
    public function setProduit($Produit): void
    {
        $this->Produit = $Produit;
    }

    /**
     * @return mixed
     */
    public function getQuantite()
    {
        return $this->Quantite;
    }

    /**
     * @param mixed $Quantite
     */
    public function setQuantite($Quantite): void
    {
        $this->Quantite = $Quantite;
    }

    /**
     * @return mixed
     */
    public function getTotal()
    {
        return $this->Total;
    }

    /**
     * @param mixed $Total
     */
    public function setTotal($Total): void
    {
        $this->Total = $Total;
    }




}
